src/Umbraco.Cms.Integrations.Crm.ActiveCampaign.Core/Constants.cs

```diff
@@ -9,6 +9,8 @@ public class Constants
 
         public const string FormsHttpClient = "FormsClient";
 
+        public const int DEFAULT_PAGE_SIZE = 10;
```

Comment:
Pascal case - `DefaultPageSize` - is the standard for constants in the code base.